app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -135,11 +142,29 @@ private fun HomeScreenContent(
                         )
                     }
                 } else {
-                    items(trendingCoins) { coin ->
+                    itemsIndexed(trendingCoins) { index, coin ->
+                        if (index >= trendingCoinsLastIndex) {
+                            SideEffect {
+                                Timber.d("onTrendingCoinsLoadMore at index: $index, lastIndex: $trendingCoinsLastIndex")
+                                onTrendingCoinsLoadMore?.invoke()
+                            }
+                        }
```

Comment:
Then we don't apply threshold to trigger loading more, right? 😄

Reply:
@hoangnguyen92dn We can define a threshold value here, but I will set it to default to 0. It depends, if we need to load more values before reaching to the last item, we can set the threshold value to 1, 2 or 3, in that case, we can not see the loading progress for load more clearly. Let me set a threshold value to 0.